include/nuttx/net/netdev.h

```diff
@@ -242,7 +242,7 @@ struct net_driver_s
 
   /* Drivers interface flags.  See IFF_* definitions in include/net/if.h */
 
-  uint8_t d_flags;
+  uint32_t d_flags;
```

Comment:
I just want to point out that in `struct ifreq` (declared in `include/net/if.h`), `ifru_flags` is defined as uint8_t. 

In `netdev_ifr_ioctl()` (in `net/netdev/netdev_ioctl.c`), when processing SIOCGIFFLAGS, we have this assignment:

```
      case SIOCGIFFLAGS:  /* Gets the interface flags */
        {
          dev = netdev_ifr_dev(req);
          if (dev)
            {
              req->ifr_flags = dev->d_flags;
            }

          ret = OK;
        }
        break;
```

So flags will be truncated.

Is this intentional?

Reply:
> 
Thank you for pointing this out.